### PR TITLE
chore: Analytics examples moved to Data on EKS repo

### LINKED ISSUE #1000: [NOTICE] Analytics  & AI/ML examples are now moving to https://awslabs.github.io/data-on-eks
## Analytics & AI/ML examples migration to https://github.com/awslabs/data-on-eks

We would like to inform you all that we will be **deprecating** the support for the following examples from this repo. 

- examples/analytics -> https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/analytics

- examples/ai-ml -> https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/ai-ml/ray

We have launched a new Open Source repo called [Data on EKS(DoEKS)](https://github.com/awslabs/data-on-eks) mainly focused on Data patterns and expand this to a bigger project. We have migrated the above examples to this new repo.  

- Get involved -> https://github.com/awslabs/data-on-eks
- Website -> https://awslabs.github.io/data-on-eks/ 

Please raise any issues/discussions under this new DoEKS repo

### PR DESCRIPTION
**Pull request by vara-bonthu**
### What does this PR do?

The following examples have been moved to this new repo. 

1/ [airflow-on-eks](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/analytics/airflow-on-eks) -> [self-managed-airflow](https://github.com/awslabs/data-on-eks/tree/main/schedulers/self-managed-airflow)
2/ [spark-k8s-operator](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/analytics/spark-k8s-operator) -> [spark-k8s-operator](https://github.com/awslabs/data-on-eks/tree/main/analytics/spark-k8s-operator)
3/ [managed-workflow-apache-airflow](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/managed-workflow-apache-airflow)-> [managed-airflow-mwaa](https://github.com/awslabs/data-on-eks/tree/main/schedulers/managed-airflow-mwaa)

The following examples will be moved in next PR once the AWS Blogs are updated with the new repo.

1/ [emr-on-eks](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/analytics/emr-on-eks)
2/ [emr-eks-fsx-lustre](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/analytics/emr-eks-fsx-lustre)
3/ [ray](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/ai-ml/ray)

- Related issue https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/1000

### Motivation

- Resolves #1000 

All the Data on EKS related Terraform patterns is maintained under new repo  https://github.com/awslabs/data-on-eks 

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs require a new example and/or doc page. In general:
- Use an existing example when possible to demonstrate a new addons usage
- A new docs page under `docs/add-ons/*` is required for new a new addon

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by Zvikan** [comment]
We should also fix our docs and GH workflows for these examples

**Comment by vara-bonthu** [comment]
> We should also fix our docs and GH workflows for these examples

Good point. I have updated the Github workflows. I don't see document references. Please do let me know if i miss any

I have also added fixes to the following github workflow deprecation issues

- The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
- Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: aws-actions/configure-aws-credentials, aws-actions/configure-aws-credentials

File: .github/workflows/plan-examples.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
         id: dirs
         run: |
           DIRS=$(python3 .github/scripts/plan-examples.py)
-          echo "::set-output name=directories::$DIRS"
```
Comment (vara-bonthu) [inline .github/workflows/plan-examples.yml:33]: `set-output` is getting deprecated soon

File: .github/workflows/plan-examples.yml [hunk 2]
```diff
@@ -75,7 +75,7 @@ jobs:
               - '*.tf'
 
       - name: Configure AWS credentials from Test account
-        uses: aws-actions/configure-aws-credentials@v1
```
Comment (vara-bonthu) [inline .github/workflows/plan-examples.yml:78]: Node.js 12 actions are getting deprecated

**Review by askulkarni2 — APPROVED** [review]

**Pull request merged**